frontend/src/metabase/static-viz/components/XYChart/utils/ticks.ts

```diff
@@ -111,7 +111,9 @@ export const getXTickProps = (
 
   const textBaseline = Math.floor(tickFontSize / 2);
   const transform = shouldRotate
-    ? `rotate(-45, ${x} ${y}) translate(${textBaseline}, 0)`
+    ? `rotate(-90, ${x} ${y}) translate(${textBaseline}, ${Math.floor(
+        tickFontSize / 3,
```

Comment:
using `tickFontSize / 3` makes the x-axis label stay right in the middle of the tick line.

#### font-size: 11px
![image](https://user-images.githubusercontent.com/1937582/187713060-573271f3-a2f4-4b21-ab84-f0c8282500f2.png)

#### font-size: 20px
![image](https://user-images.githubusercontent.com/1937582/187713175-a21146b5-7c62-496b-8a5e-9ab22934c015.png)